README.md

```diff
@@ -137,6 +137,20 @@ pub enum DataStoreError {
   }
   ```
 
+- If a field is `#[from]` and `#[backtrace]`, the Error trait's `backtrace()`
```

Comment:
`#[from]` is not necessary. It should be sufficient for a field to be `#[source]` (or just be named `source`, which is implicitly `#[source]`).